src/launchUnelevated.js

```diff
@@ -25,4 +25,10 @@ if (objShellWindows != null)
             break;
         }
     }
+
+    if (objShellWindows.count === 0) {
+        // If no File Explorer windows are open fall-back to this alternative way of executing the command
+        var item = objShellWindows.FindWindowSW(0, 0, /*SWC_DESKTOP*/8, 0, /*SWFO_NEEDDISPATCH*/1);

```

Comment:
How come some of the parameters deserve an annotation, but the rest not?

Maybe this [link](https://msdn.microsoft.com/en-us/library/windows/desktop/cc836568(v=vs.85).aspx) would help adding annotations for all of them.